tests/golden/copy-command/copy-command.bats

```diff
@@ -313,3 +313,26 @@ EOF
     b - [2000-01-01 01:01:01]
 EOF
 }
+
+@test "equal paths copy from one backend to the other" {
+  coffer create /a/b
+
+  run coffer copy /a second#/a
```

Comment:
Hmm. Maybe we need the test like this
````
coffer copy vault-local#/a vault-local#/a -f
````

Reply:
Done. Btw, this test showed that I forgot to include backend names when showing paths in the error message. Now it is fixed